metrics/open_pull_requests.py

```diff
@@ -0,0 +1,47 @@
+import csv
+import sys
+import cortx_community
+
+repo_name = sys.argv[1]
```

Comment:
You should use import argparse instead. Check out cortx_people.py for usage.

Reply:
I have changed to argparse, please review.